UIKitDZ/FirstViewController.swift

```diff
@@ -0,0 +1,248 @@
+    }
+    
+    // MARK: - Create enableShadowSettingSwitch
+    private func createEnableShadowSettingSwitch() {
+        enableShadowSettingSwitch = UISwitch(frame: CGRect(x: 290, y: 0, width: 30, height: 40))
+        enableShadowSettingSwitch.center.y = enableShadowSettingLabel.center.y
+        view.addSubview(enableShadowSettingSwitch)
+        enableShadowSettingSwitch.addTarget(self, action: #selector(shouldHideTheShadowSlider), for: .valueChanged)
+    }
+    
+    // MARK: - Create action for enableShadowSettingSwitch
+    @objc private func shouldHideTheShadowSlider(sender: UISwitch) {
+        if sender.isOn {
+            shadowOffsetSlider.isHidden = false
+        } else {
+            shadowOffsetSlider.isHidden = true
+        }
+    }
+    
+    // MARK: - Create shadowOffsetSlider
+    private func createSwadowOffsetSlider() {
+        shadowOffsetSlider = UISlider(frame: CGRect(x: 0, y: 645, width: 300, height: 40))
+        shadowOffsetSlider.center.x = view.center.x
+        shadowOffsetSlider.minimumValue = 0
+        shadowOffsetSlider.maximumValue = 10
+        shadowOffsetSlider.isHidden = true
+        view.addSubview(shadowOffsetSlider)
+        shadowOffsetSlider.addTarget(self, action: #selector(changeShadowOffset), for: .valueChanged)
+    }
+    
+    // MARK: - Add action to enableShadowSettingSwitch that changes the offset of the shadow
+    @objc private func changeShadowOffset(sender: UISlider) {
+        if sender.value == 0 {
+            testTextLabel.layer.shadowColor = UIColor(named: String(textColorPickerView.selectedRow(inComponent: 0)))?.cgColor
+            testTextLabel.layer.shadowRadius = 0
+            testTextLabel.layer.shadowOpacity = 0
+            testTextLabel.layer.shadowOffset =  CGSize(width: Int(sender.value), height: Int(sender.value))
+            testTextLabel.layer.masksToBounds = false
+        } else {
+            testTextLabel.layer.shadowColor = UIColor(named: String(textColorPickerView.selectedRow(inComponent: 0)))?.cgColor
+            testTextLabel.layer.shadowRadius = 4.0
+            testTextLabel.layer.shadowOpacity = 1.0
+            testTextLabel.layer.shadowOffset =  CGSize(width: Int(sender.value), height: Int(sender.value))
+            testTextLabel.layer.masksToBounds = false
+        }
+    }
+}
+
+// MARK: - Create data source of picker views
+extension FirstViewController: UIPickerViewDataSource {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        return 8
+    }
+}
+
+// MARK: - Create UIPickerViewDelegate extension
+extension FirstViewController: UIPickerViewDelegate {
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        switch pickerView {
+        case numberOfLinesPickerView: return String(row)
```

Comment:
```suggestion
        case numberOfLinesPickerView:
        return String(row)
```
после кейса обработка условия обычно делается со следующей строки, как изначально и предлагает Xcode